client/packages/system/src/Name/apiModern/operations.graphql

```diff
@@ -24,3 +24,31 @@ query contacts($nameId: String!, $storeId: String!) {
     }
   }
 }
+
+fragment PurchaseOrders on PurchaseOrderNode {
+  id
+  number
+  createdDatetime
+  confirmedDatetime
+  status
+  targetMonths
+  expectedDeliveryDatetime
+  comment
+  lines {
+    totalCount
+  }
+  supplier {
+    name
+  }
+}
+
+query purchaseOrders($storeId: String!) {
```

Comment:
We need to filter the purchase orders so only the ones that are for the relevent name are returned, otherwise you'll see all the purchaser orders under every suppliers record rather than just the ones for that supplier...